### PR TITLE
M3-4687 Move PrimaryNav back to the side

### PR DESCRIPTION
**Pull request by johnwcallahan**
## Description

To do this, I re-copied what was in `PrimaryNav.tsx` into `PrimaryNav_CMR.tsx` and made the following adjustments:

1. Group nav items (denoted by margin)
2. Replace hover state with gradient

I also adjusted the main content spacing to work with the new (old) layout.

**Notes:**

- I'm waiting on the designs for final spacing/placement + logos. This can be reviewed/merged as-is, though.
- The preferences "gear icon" will eventually be moved somewhere else, but I left it in for now so we can easily toggle dark theme.

Screen Shot 2020-12-02 at 2 44 50 PM

### CODE REVIEW

File: packages/manager/src/components/PrimaryNav/PrimaryNav_CMR.tsx [hunk 1]
```diff
@@ -125,416 +97,377 @@ export const PrimaryNav: React.FC<PrimaryNavProps> = props => {
...
-              setAnchorEl(undefined);
-            }}
-            getContentAnchorEl={undefined}
-            PaperProps={{ square: true, className: classes.paper }}
-            anchorOrigin={{ vertical: 'bottom', horizontal: 'left' }}
+            <ListItemText
+              primary="My Profile"
+              disableTypography={true}
+              className={classNames({
+                [classes.linkItem]: true
```
Comment (tiffwong) [inline packages/manager/src/components/PrimaryNav/PrimaryNav_CMR.tsx:318]: If these are always true, we can probably simplify these

**Review by tiffwong — APPROVED** [review]
Other than the one commend I had about cleanup, everything else looks great!

**Review by Jskobos — CHANGES_REQUESTED** [review]
Looks good.
- VLANS should be VLANs
- What's up with Images? (you said spacing wasn't final, just curious)
- DW added prefetch logic for OBJ clusters, which seems lost in translation.

**Comment by johnwcallahan** [comment]
> What's up with Images? (you said spacing wasn't final, just curious)

@Jskobos you mean the space between "Images" and "Domains"? That space is a way to group items in the Nav (also reflected in the 2D PrimaryLink array). It makes more sense when Managed is in the picture: 

Screen Shot 2020-12-03 at 9 57 46 AM

Maybe when Managed is _not_ in the picture there should be more space between the Logo and Linodes, or make "Linodes" its own category.

**Review by DevDW — APPROVED** [review]
LGTM 👍 

In Zeplin it looks like the nav item titles aren't bold?

This can be considered later, but the spacing to group the nav items does strike me as a bit odd, maybe it'd look better if the first item in each group only had a bottom border instead of a top border?

**Review by Jskobos — APPROVED** [review]

**Pull request merged**